src/commands/init.js

```diff
@@ -30,9 +31,9 @@ const templates = [{
  *
  * @returns {Promise} - Promise for the command.
  */
-export default function init({ parsedArguments, parsedOptions }) {
+export default function init({ parsedArguments, parsedOptions, directory }) {
     const { list, force } = parsedOptions.options;
-    const { name, template, version } = parsedArguments.arguments;
+    const { template, version } = parsedArguments.arguments;
```

Comment:
Name should have been used when `roc new` was invoked and seems that the variable was consumed here https://github.com/rocjs/roc/pull/109/files#diff-03189079b1b3d77ef73e9e3fdb324d6dL45